application/app/src/main/java/uk/nhs/digital/uec/api/controller/FuzzyServiceSearchController.java

```diff
@@ -39,89 +44,93 @@
+     * @return {@link ApiResponse}
+     * @throws NotFoundException
+     * @throws InvalidParameterException
+     */
+    @GetMapping("/services/byfuzzysearch")
+    @CrossOrigin(origins = "*")
+    @PreAuthorize("hasAnyRole('FUZZY_API_ACCESS')")
+    public ResponseEntity<ApiResponse> getServicesByFuzzySearch(
+            @ApiParam(SEARCH_CRITERIA_DESC) @RequestParam(name = "search_term", required = false) List<String> searchCriteria,
+            @ApiParam(SEARCH_POSTCODE_DESC) @RequestParam(name = "search_postcode", required = false) String searchPostcode,
+            @ApiParam(SEARCH_LATITUDE_DESC) @RequestParam(name = "search_latitude", required = false) String searchLatitude,
+            @ApiParam(SEARCH_LONGITUDE_DESC) @RequestParam(name = "search_longitude", required = false) String searchLongitude,
+            @ApiParam(DISTANCE_RANGE_DESC) @RequestParam(name = "distance_range", required = false, defaultValue = "25mi") String distanceRange,
+            @ApiParam(hidden = true) @RequestParam(name = "max_num_services_to_return_from_es", required = false) Integer maxNumServicesToReturnFromEs,
+            @ApiParam(MAX_NUM_SERVICES_DESC) @RequestParam(name = "max_number_of_services_to_return", required = false) Integer maxNumServicesToReturn,
+            @ApiParam(FUZZ_LEVEL_DESC) @RequestParam(name = "fuzz_level", required = false) Integer fuzzLevel,
+            @ApiParam(NAME_PRIORITY_DESC) @RequestParam(name = "name_priority", required = false) Integer namePriority,
+            @ApiParam(ADDRESS_PRIORITY_DESC) @RequestParam(name = "address_priority", required = false) Integer addressPriority,
+            @ApiParam(POSTCODE_PRIORITY_DESC) @RequestParam(name = "postcode_priority", required = false) Integer postcodePriority,
+            @ApiParam(PUBLIC_NAME_PRIORITY_DESC) @RequestParam(name = "public_name_priority", required = false) Integer publicNamePriority)
+            throws NotFoundException, InvalidParameterException {
+        log.info("Incoming request param - postcode: {}", searchPostcode);
+        log.info("Incoming request param - search_term: {}", searchCriteria);
+        log.info("Incoming request param - search_latitude: {}, search_longitude:{}", searchLatitude,searchLongitude);
+        utils.configureApiRequestParams(
+                fuzzLevel,
+                null, // filterReferralRole will be implemented in future
+                maxNumServicesToReturnFromEs,
+                maxNumServicesToReturn,
+                namePriority,
+                addressPriority,
+                postcodePriority,
+                publicNamePriority);
 
-    final ApiSuccessResponse response = new ApiSuccessResponse();
-    final ApiSearchResultsResponse searchResultsResponse = new ApiSearchResultsResponse();
+        final ApiSearchParamsResponse searchParamsResponse = new ApiSearchParamsResponse.ApiSearchParamsResponseBuilder()
+                .searchCriteria(searchCriteria == null || searchCriteria.isEmpty()
```

Comment:
Remove the search criteria check and just pass through the searchCriteria object